pkg/sink/sink_test.go

```diff
@@ -153,18 +153,18 @@ func TestHandleEvent(t *testing.T) {
 			Kind:       "PipelineResource",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "$(params.name)",
+			Name:      "$(tt.params.name)",
```

Comment:
We should probably leave some of these tests using the old params syntax to ensure backwards compatibility, since this is the first place where we are actually using the value outside of validation.

Reply:
+1 to having some tests with the old `$(params)`